js/main.js

```diff
@@ -32,6 +36,92 @@ function entryFormData(event) {
   $form.reset();
+
+  var li = document.createElement('li');
+  li.setAttribute('class', 'row margin-bottom-li');
+
+  var imgDiv = document.createElement('div');
+  imgDiv.setAttribute('class', 'column-half');
+  var imgView = document.createElement('img');
+  imgView.setAttribute('class', 'cover img-container-view');
+  imgView.setAttribute('src', data.entries[0].imgURL);
+  imgView.setAttribute('alt', 'journal-entry-image');
+
+  var textDiv = document.createElement('div');
+  textDiv.setAttribute('class', 'column-half');
+  var h2Title = document.createElement('h2');
+  h2Title.textContent = data.entries[0].title;
+  var pNotes = document.createElement('p');
+  pNotes.textContent = data.entries[0].notes;
+
+  imgDiv.appendChild(imgView);
+  textDiv.appendChild(h2Title);
+  textDiv.appendChild(pNotes);
+  li.appendChild(imgDiv);
+  li.appendChild(textDiv);
+  ul.prepend(li);
+
+  $divEntryForm.className += ' ' + 'hidden';
+  $divEntries.className = 'container';
+  data.view = 'entries';
 }
 
 $form.addEventListener('submit', entryFormData);
+
+function journalEntryView(event) {
+  if (data.view === 'entries') {
+    $divEntryForm.className += ' ' + 'hidden';
+    $divEntries.className = 'container';
+  } else if (data.view === 'entry-form') {
+    $divEntries.className += ' ' + 'hidden';
+    $divEntryForm.className = 'container';
+  }
+
+  for (var i = 0; i < data.entries.length; i++) {
+    var li = document.createElement('li');
+    li.setAttribute('class', 'row margin-bottom-li');
+
+    var imgDiv = document.createElement('div');
+    imgDiv.setAttribute('class', 'column-half');
+    var imgView = document.createElement('img');
+    imgView.setAttribute('class', 'cover img-container-view');
+    imgView.setAttribute('src', data.entries[i].imgURL);
+    imgView.setAttribute('alt', 'journal-entry-image');
+
+    var textDiv = document.createElement('div');
+    textDiv.setAttribute('class', 'column-half');
+    var h2Title = document.createElement('h2');
+    h2Title.textContent = data.entries[i].title;
+    var pNotes = document.createElement('p');
+    pNotes.textContent = data.entries[i].notes;
+
+    imgDiv.appendChild(imgView);
+    textDiv.appendChild(h2Title);
+    textDiv.appendChild(pNotes);
+    li.appendChild(imgDiv);
+    li.appendChild(textDiv);
+    ul.appendChild(li);
+  }
+}
+
+window.addEventListener('DOMContentLoaded', journalEntryView);
+
+function goToViewEntries(event) {
+  if (event.target.textContent === 'Entries') {
+    $divEntryForm.className += ' ' + 'hidden';
+    $divEntries.className = 'container';
+    data.view = 'entries';
+  }
+}
+
+$nav.addEventListener('click', goToViewEntries);
+
+function goToCreateEntries(event) {
+  if (event.target.getAttribute('name') === 'newButton') {
+    $divEntries.className += ' ' + 'hidden';
+    $divEntryForm.className = 'container';
+    data.view = 'entry-form';
+  }
+}
+
+document.addEventListener('click', goToCreateEntries);
```

Comment:
You would benefit from having one function named something like `switchView` that takes in a string that is the name of a view as its argument, then show that view and hides all other views. That way you don't need to create an individual function for each new view to navigate to it.